packages/theme/src/theme/_forms.scss

```diff
@@ -301,8 +322,14 @@ form {
 
 		.error-detail {
 			padding-left: 0;
+			margin-top: -10px;
+			margin-bottom: 0;
 
 			.text-danger {
+				line-height: 15px;
```

Comment:
`line-height` was `14px` I guess (`line-height`) is there a real difference?

Reply:
makes a minor difference when the error message occupies more than one line, this was requested by Marc

Reply:
OK. Could you make sure it's reflected on the guidelines?

Reply:
yes, it is already mentioned in the guidelines, Ref: "sizes" section under http://guidelines.talend.com/document/92132#/ui-controls/input-field